tests/unit/test_analysis_package.py

```
@@ -0,0 +1,39 @@
+"""Tests for the analysis package structure and imports.
+
+Verifies that the analysis widget package is properly organized and
+all public/private imports work correctly after the refactor to
+src/gui/widgets/analysis/.
+"""
+
+import pytest
```

Flake8 (7.3.0)

[error] 'pytest' imported but unused

(F401)